### ISSUE TITLE
Undefined variable $style_css

### ISSUE THREAD

**Issue by Zaak22** [crocodic-studio/crudbooster#1734]
I have installed crudbooster and every thing went fine while installing but when I logged in using admin@crudbooster.com I got this error 
I am using laravel 8 and php 7.4 

```
ErrorException
Undefined variable $style_css (View: D:\laravel\testphp\vendor\crocodicstudio\crudbooster\src\views\admin_template.blade.php) 
```

**Comment by grzegorzsurowiec** [comment]
I have exactly the same problem.
Clean installation of Laravel v8.77.1 (PHP v8.0.13)

**Comment by cristianoidelfonso** [comment]
I'm having the same problem here.
Using Laravel v8.78.1 (PHP v8.0.11)

**Comment by yorizpra** [comment]
i'm having same problem using php 8.0.113 & laravel 8.78.1 did anyone could solve it without downgrading php and laravel version?

**Comment by ikevinsolomon** [comment]
@fherryfherry dear friend can you please suggest a solution for this we are using laravel v8, humble request.

**Comment by MikePonce-UwU** [comment]
> 

Hey, for real?

**Comment by Zaak22** [comment]
I solved this problem by using php 7.1^ 

P.s : If you are using PHPStorm make sure to close it and reopen it
